package.json

```diff
@@ -31,18 +31,22 @@
   },
   "homepage": "https://github.com/wikimedia/restbase-mod-table-spec",
   "dependencies": {
-    "core-js": "^2.0.3",
+    "core-js": "^2.5.7",
```

Comment:
Don't need this one anymore. core-js contained shims that provided newer JS features to older node versions (mainly newer functions into object prototypes). We don't use thos older versions anymore, so no need to require this one.